public/data/Kendall_Does_Coding.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "Kendall Pinto",
+  "type": "personal",
+  "bio": "Hello. My name is Kendall. I am a Software Developer / Coder. I have made a bunch of programs in various languages, like Python, Javascript, and the one I'm using to make this website, HTML & CSS.",
+  "avatar": "https://i.ibb.co/5LsjwVB/Avatar-Maker.png",
+  "links": [
+    {
+      "name": "Check out my personal website",
+      "url": "https://kendalldoescoding.gq/",
+      "icon": "globe"
+    },
+    {
+      "name": "Take a look at my GitHub",
+      "url": "https://github.com/kendalldoescoding",
+      "icon": "github"
+    },
+    {
+      "name": "Take a look at my GitLab",
+      "url": "https://gitlab.com/kendalldoescoding",
+      "icon": "https://about.gitlab.com/images/press/press-kit-icon.svg"
```

Comment:
That...won't work as excepted 😅 we don't use icons that way.
As I said above, you can use another one such as `link`.

Reply:
still can't understand ya @Panquesito7 lol

Reply:
can you co-author the commit?

Reply:
Or you can write “gitlab” and it’ll default to globe until the gitlab icon gets added.

Reply:
> Or you can write “gitlab” and it’ll default to globe until the gitlab icon gets added.

Done! Hope this works! Will mark this has resolved.